app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -183,6 +183,7 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
                           <Row>
                             <Col className="checkBoxColumn" alignItems="center">
                               <Checkbox
+                                onClick={(e) => e.stopPropagation()}
```

Comment:
Will this have a very narrow focus?  One of the issues users have expressed is that they accidently click on the row and it makes it hard to select content.

Reply:
As far as I tested works good. But for sanity check, I moved the stop propagation to the column. So now the user have a little room for error.